root.json

```diff
@@ -12,6 +12,7 @@
     "Dropbox": "dropbox.json",
     "DuckDNS": "duckdns.json",
     "Duplicati2-canary": "duplicati2-canary.json",
```

Comment:
@FroggyFlox 
Shouldn't we be deleting this line, given we are also removing it's associated json file?
```suggestion
```